pkg/apis/flows/v1/sequence_lifecycle_test.go

```diff
@@ -440,9 +452,17 @@ func TestSequencePropagateSetAddress(t *testing.T) {
 			if got.URL != nil {
 				gotAddress = got.URL.String()
 			}
+
 			if diff := cmp.Diff(tt.wantAddress, gotAddress); diff != "" {
 				t.Error("unexpected address.url (-want, +got) =", diff)
 			}
+			gotAudience := ""
+			if got.Audience != nil {
+				gotAudience = *got.Audience
+			}
```

Comment:
when you make `wantAudience` a pointer, you don't need this here IIUC.

Reply:
Sure changed wantAudience as a pointer .